### DISCUSSION TITLE
Web interfaces

### DISCUSSION THREAD

**Discussion by Konycz** [softwarecrash/Solar2MQTT-ESP8266#93] [Q&A] [answered]
### Is your feature request related to a problem? Please describe

Is there any possibility to grant a www interface?  It's about: I have a device in the garden and I want to control the switch off via the arduino according to the state of the batteries, i.e. when it has a 25V battery, the switch will turn on, as soon as it has a 27V battery, the switch will turn off.  It would be enough if the device had a web interface where I could load the current values.  Now I have it through HA and tuya, but what if the internet turns off, it wouldn't work.

### Describe the solution you'd like

. 

### Describe alternatives you've considered

. 

### Additional context

.

**Comment by softwarecrash** [comment]
Please Take a Look at the wiki for possible ways to Set or get data

**Comment by Konycz** [comment]
Where exactly?  I guess I'm still overlooking it :)

**Comment by softwarecrash** [comment]
https://github.com/softwarecrash/Solar2MQTT/wiki/

**Comment by Konycz** [comment]
Yes, I know where the wiki is, but where and how to set it up so that I can read the data from the second device on the local network, when the mqtt server is elsewhere and when the internet goes down, I can't access it.  I really need to read the values ​​even when the internet is not working

**Comment by softwarecrash** [comment]
When Internet Not working but Network avaible you can get data over esp-ip/livejson, it Delivery all Data as json String.
Unfortunaly is See its Not well documented in this project

**Comment by Konycz** [comment]
And how can this data be obtained?

**Comment by softwarecrash** [comment]
Send from another ESP or Something Network connected a http request to this Adress.

**Comment by Konycz** [comment]
I don't understand this... could I ask for an explanation?? what will the address look like to which I will connect from the second device to get the raw data in json??

**Comment by softwarecrash** [comment, marked as answer]
The Adresse ist http://esp-ip/livejson

For Test Just call this Adress in your Browser, you will get the livejson String.
So with any other connected device you need to build a http request.
Check 
https://www.arduino.cc/reference/en/libraries/httpclient/ for more details

**Comment by Konycz** [comment]
Great, thank you very much
